examples/image_classification/index.html

```diff
@@ -182,6 +195,26 @@ <h1>0%</h1>
           </div>
         </div>
       </div>
+
+      <div class='tab-content clearfix' style='display: none;' id='progressruntime'>
+        <div class='loading-page'>
+          <div class='counter'>
+            <div class='loading'>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
+              <div></div>
```

Comment:
why do we need these pairs of `<div></div>`?

Reply:
@huningxin , these are used for the light blue/blue loading animation effects.

```
.loading div {
  position: absolute;
  width: 19px;
  height: 19px;
  border-radius: 0%;
  background: rgba(25, 190, 225, 1.0);
  animation: loading 0.6s linear infinite;
}

.loading div:nth-child(1) {
  top: 0px;
  left: 0px;
  animation-delay: 0s;
}

.loading div:nth-child(2) {
  top: 0px;
  left: 20px;
  animation-delay: -0.4s;
}

.loading div:nth-child(3) {
  top: 0px;
  left: 40px;
  animation-delay: -0.8s;
}

.loading div:nth-child(4) {
  top: 20px;
  left: 0px;
  animation-delay: -0.4s;
}

.loading div:nth-child(5) {
  top: 20px;
  left: 20px;
  animation-delay: -0.8s;
}

.loading div:nth-child(6) {
  top: 20px;
  left: 40px;
  animation-delay: -1.2s;
}

.loading div:nth-child(7) {
  top: 40px;
  left: 0px;
  animation-delay: -0.8s;
}

.loading div:nth-child(8) {
  top: 40px;
  left: 20px;
  animation-delay: -1.2s;
}

.loading div:nth-child(9) {
  top: 40px;
  left: 40px;
  animation-delay: -1.6s;
}
```

Reply:
Great to know. Thanks for the education.